serve/package.go

```diff
@@ -31,6 +31,7 @@ This creates a directory with the plugin binaries, package.json and documentatio
 type PackageJSON struct {
 	SchemaVersion    int                `json:"schema_version"`
 	Name             string             `json:"name"`
+	Message          string             `json:"message"`
```

Comment:
Would this be the equivalent of a changelog? If so can we call it `Changelog`?

Reply:
It is equivalent to a changelog entry, yes. It's currently also known as `message` in the API and DB layers, associated with a particular plugin version. https://github.com/cloudquery/cloud/blob/main/internal/servergen/openapi/components/schemas/PluginVersion.yaml#L26-L29

I chose `message` because of its analogue with a commit message. Do you think it's worth the effort of updating?

Reply:
Ah if it's a single message no need to change it